### PR TITLE
M0090IDE-3546 MMM CLI documentation

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: ApplicationDeveloperGuide/mmm.rst [hunk 1]
```diff
@@ -405,26 +405,272 @@ Alternatively, the build of a MicroEJ module project can be started from the bui
...
+~~~~~~~~~~~
+
+.. _mmm.cli.commands.init:
+
+**init**
+
+The subcommand ``init`` creates a new project (executes Easyant with ``skeleton:generate`` target).
+The skeleton and project information must be passed with the following system properties:
+
+- ``skeleton.org``: organisation of the skeleton module. Defaults to ``org.apache.easyant.skeletons``.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/mmm.rst:535]: update with init

Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/mmm.rst]: Done.

**Comment by frivieremicroej** [comment]
Steps:
- review and merge https://github.com/MicroEJ/docs/pull/194
- review wording of the above comment

**Review by frivieremicroej — APPROVED** [review]
OK waiting  for SDK 5.4.0 release before merging to master

**Pull request merged**